src/schnetpack/configs/model/representation/wacsf.yaml

```diff
@@ -0,0 +1,30 @@
+_target_: schnetpack.representation.SymmetryFunctions
+radial:
+  _target_: schnetpack.representation.symfuncs.RadialSF
+  radial_basis:
+    _target_: schnetpack.nn.radial.GaussianRBFCentered
+    n_rbf: 20
+    cutoff: ${cutoff}
+  elemental_basis:
+    _target_: schnetpack.nn.symmetry_functions.WeightedElements
+  cutoff_fn:
+    _target_: schnetpack.nn.cutoff.CosineCutoff
+    cutoff: ${model.representation.radial.radial_basis.cutoff}
+angular:
+  _target_: schnetpack.representation.symfuncs.AngularSF
+  radial_basis:
+    _target_: schnetpack.nn.radial.GaussianRBFCentered
+    n_rbf: 5
+    cutoff: ${cutoff}
+  elemental_basis:
+    _target_: schnetpack.nn.symmetry_functions.WeightedElements
+  angular_basis:
+    _target_: schnetpack.nn.symmetry_functions.AngularBasis
+    zetas:
```

Comment:
perhaps in this case it is more readable to write `zetas: [1, 4, 8]`

Reply:
Changed.